include/boost/spirit/home/x3/support/traits/value_traits.hpp

```diff
@@ -9,16 +9,14 @@
 #if !defined(BOOST_SPIRIT_X3_VALUE_TRAITS_MAY_07_2013_0203PM)
 #define BOOST_SPIRIT_X3_VALUE_TRAITS_MAY_07_2013_0203PM
 
-#include <boost/utility/value_init.hpp>
-
 namespace boost { namespace spirit { namespace x3 { namespace traits
 {
     template <typename T, typename Enable = void>
     struct value_initialize
     {
         static T call()
         {
-            return boost::value_initialized<T>();
+            return {};
```

Comment:
Please file this as a separate PR. The change does not need addition tests, while other changes in this PR do.